packages/perseus/src/widgets/deprecated-standin/__tests__/deprecated-standin.test.ts

```diff
@@ -0,0 +1,93 @@
+import "@testing-library/jest-dom"; // Imports custom matchers
+
+import {testDependencies} from "../../../../../../testing/test-dependencies";
+import * as Dependencies from "../../../dependencies";
+import {renderQuestion} from "../../__tests__/renderQuestion";
+
+const question = {
+    content:
+        "Read the excerpt and answer the question below. \n\nThe Governor and Council of the Massachusetts had much conference many days; and at last . . . . concluded a peace and friendship with [[\u2603 widget 1]], upon these conditions.",
+    images: {},
+    widgets: {
+        "widget 1": {
+            graded: true,
+            version: {
+                major: 0,
+                minor: 0,
+            },
+            static: false,
+            type: "deprecated-standin",
+            options: {
+                static: false,
+            },
+            alignment: "default",
+        },
+    },
+} as const;
+
+describe("Deprecated Standin widget", () => {
+    beforeEach(() => {
+        jest.useRealTimers();
```

Comment:
Just double-checking, is this necessary for these tests to pass?  I don't see any `async` tests so I'd be surprised if real timers are needed.